src/js/newsletter-subscription.js

```diff
@@ -1,5 +1,28 @@
 const NEWSLETTER_SUBSCRIBE_URL = 'https://www.mozilla.org/en-US/newsletter/';
 
+/** Opinionated function for extracting data from the form without needing FormData
+ *
+ * @param {object} form element
+ * @returns {string} URI-encoded string
+ */
+function getPayloadString(form) {
+  if (typeof URLSearchParams === 'undefined') {
+    // if we lack URLSearchParams or a fully-featured FormData (eg: IE11)
+    // let's just manually extract the data we need
+    const formInputs = form.getElementsByTagName('input');
+    const pairs = [];
+
+    for (let i = 0; i < formInputs.length; i += 1) {
+      const field = formInputs[i];
+      pairs.push(`${field.name}=${encodeURIComponent(field.value)}`);
+    }
+    return pairs.join('&');
+    // eslint-disable-next-line no-else-return
```

Comment:
Not sure if I'm a fan of the lint rule, but in this context it might be easier to just follow it, move the return outside of else and drop the branch

Reply:
Noted, and yeah I agree. Fixed!